resdk/resolwe.py

```diff
@@ -293,16 +293,23 @@ def run(self, slug=None, input={}, descriptor=None,  # pylint: disable=redefined
         else:
             raise ValueError("Unexpected behaviour at get process with slug {}".format(slug))
 
-        # Upload files in basic:file fields
+        # Pre-process inputs
         try:
             for schema, fields in iterate_fields(input, process['input_schema']):
                 field_name = schema['name']
                 field_type = schema['type']
                 field_value = fields[field_name]
 
+                # Wrapp `list:` fields into list if they are not already
```

Comment:
I agree that we should handle exceptions better. But first thing is to prevent exceptions from happening. Isn't our platform about taking care of tasks that we can instead of user?